client/src/auth/index.js

```diff
@@ -5,10 +5,13 @@ import config from '../config'
 const { apiUrl } = config
 
 export const [useAuth, authFetch, login, logout] = createAuthProvider({
-  onUpdateToken: (token) => {
-    fetch(`${apiUrl}/auth/refresh`, {
+  onUpdateToken: async (oldToken) => {
+    const res = await fetch(`${apiUrl}/auth/refresh`, {
       method: 'POST',
-      body: token.access_token,
```

Comment:
again, we were trying to access token like it's an object, but in fact it's just a string.